keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers/README.md

```diff
@@ -235,51 +452,101 @@ _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
```

Comment:
nit: you should update to the newer KEP template

Reply:
i thought i did update to the newer template, is this a stale question?

Reply:
i see that i did not fully copy the template, i'm updating now.